lib/probes/amqplib.js

```diff
@@ -167,6 +168,8 @@ function patchConsume (proto) {
         () => cb.call(this, msg),
         conf
       )
-    }), options)
+    }
+    const f = ao.lastEvent ? ao.bind(consumePatched) : consumePatched
```

Comment:
What kind of situation results in no lastEvent?
I was expecting an additional test to cover it, maybe I'm not looking in the right place.

Reply:
The primary place I see it in users programs is during startup. Because most of what I deal with are http servers it is generally the http request that initiates the tracing. Various programs (databases and queues, etc.) tend to set things up before the http server starts listening to requests.

`ao.lastEvent` is basically a `we are tracing` check.